doc/SAI-IPv6-Segment-Routing-Update.md

```diff
@@ -477,7 +477,7 @@ typedef struct _sai_local_sid_entry_t
 
 
 ## References ##
```

Comment:
draft-filsfils-spring-srv6-net-pgm-insertion-04 is missing (H.Insert, H.Insert.Red, End.B6.Insert, End.B6.Insert.Red)